server/subsonic/helpers.go

```diff
@@ -149,7 +153,7 @@ func getTranscoding(ctx context.Context) (format string, bitRate int) {
 
 // This seems to be duplicated, but it is an initial step into merging `engine` and the `subsonic` packages,
 // In the future there won't be any conversion to/from `engine. Entry` anymore
-func childFromMediaFile(ctx context.Context, mf model.MediaFile) responses.Child {
+func ChildFromMediaFile(ctx context.Context, mf model.MediaFile) responses.Child {
```

Comment:
This function has no need to be published anymore, right?

Reply:
You're right:
[jukebox-mode c1c25ff6] Reduce visibility of ChildFromMediaFile